silhouette/app/com/mohiva/play/silhouette/impl/providers/OAuth2Provider.scala

```diff
@@ -142,12 +145,16 @@ trait OAuth2Provider extends SocialProvider with OAuth2Constants with Logger {
    * @return The info containing the access token.
    */
   protected def getAccessToken(code: String)(implicit request: RequestHeader): Future[OAuth2Info] = {
-    httpLayer.url(settings.accessTokenURL).withHeaders(headers: _*).post(Map(
+    val params = Map(
```

Comment:
I did run the formatting script as mentioned in the contributing guide, not sure why this wasn't corrected by it. Any idea, what I am doing it wrong?